tests/test_adjoint_jacobian.py

```diff
@@ -462,15 +475,13 @@ def circuit(params):
             qml.Rot(params[1], params[0], 2 * params[0], wires=[0])
             return qml.expval(qml.PauliX(0))
 
-        spy_numeric = mocker.spy(qml.tape.JacobianTape, "numeric_pd")
```

Comment:
Should we be checking this?

Reply:
It seems this is not required anymore as they are also deprecated in PL test-suite.

Reply:
Yep, `JacobianTape` (and all the other tape subclasses!) are being deprecated in favour of the functions in the `qml.gradients` subpackage.

Reply:
Note that these checks _could_ be updated here, _but_ I don't think it makes sense to unittest PL core functionality in the plugin test.